pkg/daemon/outputs.go

```diff
@@ -27,10 +29,19 @@ func (srv *Daemon) outputsHandler(engine api.Engine) func(w http.ResponseWriter,
 
 		tgw := rpc.NewOutputWriter(w, r)
 
-		err = engine.DoCollectOutputs(r.Context(), req.Runner, req.RunID, tgw)
+		result := false
+		defer tgw.WriteResult(result)
+
+		var file bytes.Buffer
+		fw := bufio.NewWriter(&file)
+
+		err = engine.DoCollectOutputs(r.Context(), req.Runner, req.RunID, tgw, fw)
 		if err != nil {
-			log.Errorw("collect outputs error", "err", err.Error())
+			log.Warnw("collect outputs error", "err", err.Error())
```

Comment:
Demoting this to `Warnw` because this happens when we request a non-existing run id.